src/ca/mcgill/cs/jetuml/views/StringViewer.java

```diff
@@ -41,7 +41,7 @@
  * @author Martin P. Robillard.
  * @author Kaylee I. Kutschera - Migration to JavaFX
  */
-public final class StringViewer
+public final class StringViewer implements ApplicationFont

```

Comment:
Please use static imports instead of interface extension, to preserve the integrity of the subtyping relation